Remotion/ObjectBinding/Web.IntegrationTests/ObjectBinding.Web.IntegrationTests.csproj

```diff
@@ -31,6 +31,9 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Coypu, Version=3.0.0.0, Culture=neutral, PublicKeyToken=null">
+      <HintPath>..\..\..\packages\Coypu.3.1.1\lib\net45\Coypu.dll</HintPath>
+    </Reference>

```

Comment:
Did not find places where this reference is needed, addition might have been an accident.

Reply:
You're right that I just mindlessly slammed alt + enter, but the `IsFirefox()` check does require Coypu. I've gone ahead and added a proper Nuget package reference to the project